libs/features/webapp-prompt/webapp-prompt.css

```diff
@@ -138,6 +138,10 @@
     left: 12px;
   }
 
+  .appPrompt-close:focus {
+    outline: 2px solid #157AF3;
```

Comment:
Looks good to me, however I'm wondering whether we should use variables for colours. Maybe @overmyheadandbody has additional feedback on this.
Also, one thing I noticed: the focus order is "cancel" button, then "X", even though the "X" mark is higher than the "cancel" button. Is this intended?

Reply:
The focus order should be indeed different, but that's addressed in [another pending PR](https://github.com/adobecom/milo/pull/1853)

Reply:
We don't set any other focus colors on any of the Gnav elements. This is intentional, as we want all page elements to be consistent in their focused state. Since a lot of other elements don't have an explicit value, it means we ultimately rely on the browser to set this color, through its default `:focus-visible` value. Setting an explicit value might break this consistency for this particular element. Is there no other way of handling this issue, but maintain the user agent defaults?

Reply:
@overmyheadandbody I'll investigate further and see if there's a way without setting the focus color 👍

Reply:
I found a solution that preserves the user agent focus color, it's a bit off on safari, but it's better than what we had before